res/code/_get-version.php

```diff
@@ -24,8 +24,8 @@ function getVersion($platform, $level) {
         // For now, we use alpha from local instance of s.keyman.com because 
         // downloads.keyman.com may be out of sync with s.keyman.com for alpha.
         // As we stablise the CI server, we may be able to eliminate this
-        $json = file_get_contents($this->versionJsonFilename);
-        if($json === NULL) {
+        $json = @file_get_contents($this->versionJsonFilename);
+        if($json === FALSE) {
```

Comment:
This comparison was previously always false ... `file_get_contents` returns `FALSE` on failure, not `NULL`.